### ISSUE TITLE
Unable start tunnel on Windows

### ISSUE THREAD

**Issue by danvin2** [danielpaulus/go-ios#481]
It works perfectly on mac, but when trying to start tunnel on windows I'm getting this:
` C:\goios> ./ios.exe tunnel start
{"level":"warning","msg":"go-ios agent is not running. You might need to start it with 'ios tunnel start' for ios17+. Use ENABLE_GO_IOS_AGENT=yes for experimental daemon mode.","time":"2024-09-21T16:04:35-04:00"}
{"level":"info","msg":"Tunnel server started","time":"2024-09-21T16:04:35-04:00"}
{"error":"UpdateTunnels: failed to get list of devices: dial tcp 127.0.0.1:27015: connectex: No connection could be made because the target machine actively refused it.","level":"warning","msg":"failed to update tunnels","time":"2024-09-21T16:04:36-04:00"}
{"error":"UpdateTunnels: failed to get list of devices: dial tcp 127.0.0.1:27015: connectex: No connection could be made because the target machine actively refused it.","level":"warning","msg":"failed to update tunnels","time":"2024-09-21T16:04:37-04:00"}
{"error":"UpdateTunnels: failed to get list of devices: dial tcp 127.0.0.1:27015: connectex: No connection could be made because the target machine actively refused it.","level":"warning","msg":"failed to update tunnels","time":"2024-09-21T16:04:38-04:00"}
{"error":"UpdateTunnels: failed to get list of devices: dial tcp 127.0.0.1:27015: connectex: No connection could be made because the target machine actively refused it.","level":"warning","msg":"failed to update tunnels","time":"2024-09-21T16:04:39-04:00"}`

Running as Administrator. Device connected. Firewall turned OFF. Wintune is in system32 folder.
Please advise.
Thank you!!

**Comment by GaoXing0608** [comment]
Have you solved?

**Comment by vividius** [comment]
I had similar problem.
Solved after downgrading windows and python and doing zillion other things which is really hard to remember.
Do you still have this issue?

**Comment by gbalduzzi** [comment]
I'm having the same problem right now, is there any way to debug this?

**Comment by shamanec** [comment]
Do you have iTunes installed?

**Comment by gbalduzzi** [comment]
No, that actually solved the issue. It may be useful to add a line about this in the Readme file. 

It's obvious in hindsight but at first I was very confused and could not find informations about it

**Comment by Dasurkarfaraz** [comment]
@gbalduzzi @shamanec  I am also facing same issue can you help other to resolve issue.

**Comment by aluedeke** [comment]
yes you need usbmuxd running on windows which you get by installing [Apple Device ](https://apps.microsoft.com/detail/9np83lwlpz9k)
